ios/FluentUI/Drawer/DrawerController.swift

```diff
@@ -428,8 +425,10 @@ open class DrawerController: UIViewController {
     private var useCustomBackgroundColor: Bool = false
     /// for iPad split mode, navigation bar has a different dark elevated color, and if it is a `.down` presentation style, match `Colors.NavigationBar.background` elevated color
     private var useNavigationBarBackgroundColor: Bool = false
+    /// Drawer controller design tokens
+    private let drawerTokens = MSFDrawerTokens()
```

Comment:
How can we override the theme on the drawer?
It seems like we're only getting the default theme or the one for the current window?

Reply:
`DrawerController` conforms to `FluentWindowProvider` to reflect window theme changes. 
Currently there isn't any solution to override theme as we do in SwiftUI. There is open ticket in ADO to address this.

Reply:
Great. It's good to keep track of that. I'm not necessarily expecting the theme override to come from the view hierarchy as it does in SwiftUI Environment Values. But I believe providing a property in the drawer to set the theme would bring it to a closer level of functionality compared to the the SwiftUI Drawer.